osc/commandline.py

```diff
@@ -8342,6 +8342,36 @@ def do_whois(self, subcmd, opts):
             if len(user) == 3:
                 print("%s: \"%s\" <%s>" % (user[0], user[1], user[2]))
 
+    @cmdln.alias('cpc')
+    def do_createpbuildconfig(self, subcmd, opts, *args):
```

Comment:
Wouldn't be better use name with dashes: `create-pbuild-config` ?
There's no explicit support for that in cmdln.py, but I could add it easily.